compose.js

```diff
@@ -88,12 +92,10 @@ function mergeComposable(dstDescriptor, srcComposable) {
   combineProperty('configuration', assign);
   combineProperty('deepConfiguration', merge);
   if (Array.isArray(srcDescriptor.initializers)) {
-    dstDescriptor.initializers = srcDescriptor.initializers.reduce((result, init) => {
-      if (isFunction(init) && result.indexOf(init) < 0) {
-        result.push(init);
-      }
-      return result;
-    }, Array.isArray(dstDescriptor.initializers) ? dstDescriptor.initializers : []);
+    const initializers = (dstDescriptor.initializers || [])
```

Comment:
In this case DRY would complicate reading. If this was a real app I would DRY. But here I don't what readers to just up to `prepareInitializers` and down to this line. Also, `prepareInitializers` is not the best name. So, I'll leave it as is.

If you remove `isFunction` here some tests would fail. And those are pretty valid tests for valid use cases. So, yeah, the filtering here is necessary.